packages/forms-web-app/src/config.js

```diff
@@ -105,6 +105,20 @@ module.exports = {
 					minLength: 7,
 					minLengthMessage: 'List entry number must be 7 digits'
 				}
+			},
+			appealReferenceNumber: {
+				regex: {
+					regex: '[0-9]',
```

Comment:
Is this supposed to be just one digit?

Reply:
Oh wait does the min max thing like iterate through and check each char with this regex?

Reply:
No it should be several digits, good spot, just changed it.